docs/advanced/persistence-and-collaboration.mdx

```diff
@@ -0,0 +1,115 @@
+---
+title: Persistence and Collaboration
+---
+
+## Introduction
+
+By default `Variable` state is purely client-side - by default its source of truth is the browser memory.
+It also means that refreshing the browser page resets the `Variable` state, and that each user has their own 'copy'
+of the data. To customize the behaviour, `Variable` accepts a store property. This allow changing the `Variable`'s source of truth,
```

Comment:
we seem to be switching between referring to `Variable` without any articles and referring to it with articles ie referring to "a" or "the" `Variable` or just referring to the object as the class itself. I don't mind either way but let's probably stick with one